client-dash.php

```diff
@@ -9,6 +9,8 @@
  * Plugin Name: Client Dash
  * Description: Creating a more intuitive admin interface for clients.
  * Version: 2.1.6
+ * Requires at least: 4.7.0
```

Comment:
4.7 seems suspiciously high, but that _is_ what's in our `readme.txt` 🤔 

Would you be able to run the ZIP from the .Org directory through https://wpseek.com/pluginfilecheck/ to double-check that for me?